File: bernardo/Problema0202/problema0202.py
# Entrada de valores e verificação de números maiores que 30
quantidade = 0

for x in range(1, 16):
    num = float(input(f"Digite o {x} número: "))
    if num > 30:
        quantidade += 1

# Saída e cálculo da raíz quadrada
print(f"A quantidade de números maiores que 30 é: {quantidade}")

File: bernardo/Problema0202/problema0202.java
import java.util.Scanner;

public class problema0202 {
    static Scanner input = new Scanner(System.in);

    public static void main(String[] args) {

        // Declaração de variáveis
        float num;
        int quantidade = 0;

        // Entrada de valores e verificação de números maiores que 30
        for (int i = 0; i < 15; i++) {
            System.out.println("Digite o " + (i + 1) + " número: ");
            num = Float.valueOf(input.nextLine());
            if (num > 30) {
                quantidade++;
            }
        }

        // Saída
        System.out.println("A quantidade de números maiores que 30 é: " + quantidade);

    }
}
